app/screens/WalletNavigator/screens/CreateWallet/OnboardingNetworkSelectScreen.tsx

```diff
@@ -0,0 +1,30 @@
+import { SectionTitle } from '@components/SectionTitle'
+import { getEnvironment } from '@environment'
+import { RowNetworkItem } from '@screens/AppNavigator/screens/Settings/components/RowNetworkItem'
+import { translate } from '@translations'
+import * as React from 'react'
+import { View } from 'react-native'
+
+export function OnboardingNetworkSelectScreen (): JSX.Element {
+  const networks = getEnvironment().networks
+
+  return (
+    <View testID='onboarding_network_selection_screen'>
+      <SectionTitle
+        testID='onboarding_network_selection_screen_title'
+        text={translate('screens/OnboardingNetworkSelectScreen', 'SELECT NETWORK')}
+      />
+
+      {
+        networks.map((network, index) => (
+          <RowNetworkItem
```

Comment:
I actually prefer just make another simple component here.

1. This is not part of Settings screen anyway
2. Wish this page page auto dismiss after selected a network (my personal preference on the UX)
3. The original row component will has fixed alert message and tidier

Reply:
After consulting Jonathan, he mentioned that the network selection UI will have changes again in the future and both screen (onboarding and setting) should always be the same.

Can I still go with the current solution which is to share the same `RowNetworkItem` component, but I'll move it to `app/components` ?